app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
 
+    people = {}
```

Comment:
The `people` dictionary is used to store `Person` instances by their names. This can lead to issues if there are duplicate names in the input list, as it will overwrite previous entries. Consider using a list or another structure that can handle duplicates.